product-service/getProductById.ts

```diff
@@ -0,0 +1,24 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import 'source-map-support/register';
+import * as data from './productList.json';
+
+export const productById: APIGatewayProxyHandler = async (event, _context) => {
+  const { id } = event.pathParameters;
+  const item = data.find((item: any) => item.id === id );
+  const response = {
+    statusCode: 200,
+    headers: {
+      'Access-Control-Allow-Origin': '*',
+      'Access-Control-Allow-Credentials': true,
+    },
+    body: '',
+  };
+  try {
```

Comment:
Hi @MaksimPopov64 i have tested your endpoint trying to get Product not found error, but it seems just return an empty object with a 200 status. I think it would be better to handle it with a message or at least a 404. 
Ex: https://fon3pkjz3m.execute-api.eu-west-1.amazonaws.com/dev/product/7567ec4b-b10c-48c5-9345-fc73c48a80ab